packages/components/ng-ovh-payment-method/src/components/register/constants.js

```diff
@@ -4,22 +4,22 @@ import BANK_ACCOUNT from './assets/bank_account.png';
 
 export const DEFAULT_DISPLAY_PER_LINE = 3;
 
+export const DEFAULT_SELECTED_PAYMENT_METHOD_TYPE = 'CREDIT_CARD';
+
+export const PAYPAL_PAYMENT_METHOD = 'PAYPAL';
+
 export const DEFAULT_ORDERED_PAYMENT_METHOD_TYPES = [
-  'CREDIT_CARD',
-  'PAYPAL',
+  DEFAULT_SELECTED_PAYMENT_METHOD_TYPE,
+  PAYPAL_PAYMENT_METHOD,
   'BANK_ACCOUNT',
```

Comment:
You can create another constant for 'BANK_ACCOUNT' ? Is that a another type ?

Reply:
Sure. I my case, I don't need to use 'BANK_ACCOUNT' but some constants values were duplicated that's why I refactor this part

Reply:
@sachinrameshn It's fixed !